filehandling.c

```diff
@@ -0,0 +1,48 @@
+#include<stdio.h>
+
+struct employee
+{
+	char employeeName[50];
+	int employeeId;
+};
+
+int main()
+{
+	FILE *fp=NULL;
```

Comment:
add space before and after each operator File* fp = NULL;

Reply:
I done changes as your review commands . can you please verify once